integrationTests/factory/componentsHelper.go

```diff
@@ -52,6 +52,7 @@ func CreateDefaultConfig() *config.Configs {
 
 	p2pConfig.KadDhtPeerDiscovery.Enabled = false
 	prefsConfig.Preferences.DestinationShardAsObserver = "0"
+	prefsConfig.Preferences.ConnectionWatcherType = "print"
```

Comment:
please define constants somewhere for these values

Reply:
added them to libp2p constants and used them everywhere